### PR TITLE
TOOLS-2535: commitIndexBuild and createIndexes oplog entries should b…

### PR DESCRIPTION
**Pull request by huan-Mongo**
…uild indexes with the createIndexes command during oplog replay

### CODE REVIEW

File: mongorestore/oplog.go [hunk 1]
```diff
@@ -328,6 +350,35 @@ func convertCreateIndexToIndexInsert(op db.Oplog) (db.Oplog, error) {
 	return op, nil
 }
 
+// ConvertcommitIndexBuildToCreateIndexes converts a "commitIndexBuild" oplog entry to "createIndexBuilds" Op
+// Returns true if the operation should be applied to the destination.
+// TODO: add special handling for commitIndexBuild during oplog sync in MGOMIRROR-343
```
[the remark this answers is not in the document]
Comment (huan-Mongo) [reply mongorestore/oplog.go]: Done!

**Comment by huan-Mongo** [comment]
@tfogo I made the change to call createIndex cmd for both `commitIndexBuilds` and `createIndexes` oplog

**Comment by dragonlee8** [comment]
Make sense. I will add the logging!

**Comment by huan-Mongo** [comment]
Make sense! I will add the log.

**Pull request merged**